maraboupy/test/test_tf.py

```diff
@@ -44,7 +53,7 @@ def test_conv_mp1():
     """
     filename = os.path.join(os.path.dirname(__file__), FG_FOLDER, "conv_mp1.pb")
     network = Marabou.read_tf(filename)
-    evaluateNetwork(network, numPoints = 5) 
+    evaluateNetwork(network, numPoints = 3)
```

Comment:
Why do we reduce the number of sampled points?

Reply:
I reduced the number of points for the convolutional networks that are much larger than other test networks so that we don't add too much time to Travis. I can just reduce NUM_RAND to 5 for all tests, and the tests run in about 18 seconds.